scripts/zones/Metalworks/npcs/Hungry_Wolf.lua

```diff
@@ -20,21 +21,40 @@ require("scripts/globals/titles");
 
 function onTrade(player,npc,trade)
     if (player:getQuestStatus(BASTOK,SMOKE_ON_THE_MOUNTAIN) ~= QUEST_AVAILABLE and
-      trade:hasItemQty(4395,1) and trade:getItemCount() == 1) then
+        trade:hasItemQty(4395,1) and trade:getItemCount() == 1) then
             player:startEvent(0x01ad);
     end
---]]
 end;
 
 -----------------------------------
 -- onTrigger Action
 -----------------------------------
 
 function onTrigger(player,npc)
+    local s_table = Switch_Table[player:getLocalVar("Bait_and_Switch_Quest_Order")];
+    local item = CheckOptionForItem(player);
+    local SmokeOnTheMountain = player:getQuestStatus(BASTOK,SMOKE_ON_THE_MOUNTAIN);
+    local BnSDialogue = player:getLocalVar("Bait_and_Switch_Quest_NPCs");
+    local active = player:getLocalVar("Bait_and_Switch_Quest_Active");
+    local luto = (player:getLocalVar("Luto_Caught"));
+    local miledo = (player:getLocalVar("Miledo_Caught"));
+    local keyParam = 0;
+    if (luto == 1) then
+        keyParam = 2;
+    elseif (miledo == 1) then
+        keyParam = 1;
+    end
+    local timesUP = false;
+    if (os.time() > player:getLocalVar("Bait_and_Switch_Time_Limit") and item == 6) then
+        timesUP = true;
+    end
```

Comment:
see if you can toss this into the global since i feel ive seen this block of code around quite a bit so far

Reply:
yep, this could be global too

Reply:
fixed